src/main/java/com/liveramp/daemon_lib/executors/forking/ForkedJobletRunner.java

```diff
@@ -28,14 +33,30 @@ public static void main(String[] args) throws IOException, ClassNotFoundExceptio
     String configStorePath = args[1];
     String daemonWorkingDir = args[2];
     String id = args[3];
+    String customSerializationClasses = args.length > 4 ? args[4] : "";
 
     JobletFactory factory = (JobletFactory)Class.forName(jobletFactoryClassName).newInstance();
-    JobletConfig config = JobletConfigStorage.production(configStorePath).loadConfig(id);
     DefaultJobletStatusManager jobletStatusManager = new DefaultJobletStatusManager(daemonWorkingDir);
+    final List<Function<byte[], JobletConfig>> deserializersWithDefault = Stream.concat(
+        Stream.of(JobletConfigStorage.getDefaultDeserializer()),
```

Comment:
I think this would make more sense as the second argument (unless I'm misunderstanding the ordering with Stream.concat).  Otherwise, I imagine when you use a custom deserializer, it'll be erroring every time with the default one before trying the custom one.

Reply:
Mos def. Let me swappify.